gateway/api/models.py

```diff
@@ -58,6 +58,27 @@ def __str__(self):
         return f"{self.id}"
 
 
+class Provider(models.Model):
```

Comment:
How is the instance of the `Provider` created?  Is the new viewset necessary for the `Provider`?

Reply:
Good question. No, initially it's not going to be public. We are thinking to use the panel administrator or some kind of script in the first steps meanwhile we don't close the process to publish and manage a 3rd party function.

Reply:
I see.  I believe that as an open source project, we need to provide some ways to fill the `provider` database.

Reply:
Mmm... A mock provider sounds really good. I agree. Can we add it with internal issue #257? I think it has more sense and we don't add more changes on this one. WDTY?

Reply:
I'm  OK with a separate issue.  I would like an issue in this repo.  Thanks!